### PR TITLE
Closes #450 - Enable 'fluo classpath' command to run on Mac OS X

### LINKED ISSUE #450: Enable 'fluo classpath' command to run on Mac OS X
The command is currently a complicated bash script.  It might easier to just rewrite it in python.

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

**Comment by keith-turner** [comment]
@mikewalch I think you had mentioned offline rewriting the bash script in Python.  Was there any particular reason you decided to write in Java vs Python?

File: modules/cluster/src/main/java/io/fluo/cluster/runner/AppRunner.java [hunk 1]
```diff
@@ -195,4 +194,64 @@ public void scan(FluoConfiguration config, String[] args) {
...
+            classpath.append(",");
+          }
+          classpath.append(f.getAbsolutePath());
+        }
+      }
+    } else {
+      if (classpath.length() != 0) {
+        classpath.append(":");
+      }
+      classpath.append(libDir + "/*");
```
Comment (keith-turner) [inline modules/cluster/src/main/java/io/fluo/cluster/runner/AppRunner.java]: Should this be `libDir.getAbsolutePath()` like `f..getAbsolutePath()` above?

Comment (mikewalch) [inline modules/cluster/src/main/java/io/fluo/cluster/runner/AppRunner.java]: yes. fixed in 0398b4740

**Comment by mikewalch** [comment]
I wrote it in Java because I didn't want to add a dependency on Python for a little bit of code.  While I like using Python for `fluo-deploy`, I don't think adding Python as a dependency to Fluo is worth it unless we can move a decent amount of code to Python.

**Comment by keith-turner** [comment]
+1

**Pull request merged**